tests/trainer/logging_tests/test_eval_loop_logging_1_0.py

```diff
@@ -675,11 +670,15 @@ def on_test_epoch_start(self, trainer, pl_module):
             self.make_logging(pl_module, 'on_test_epoch_start', 3, on_steps=self.choices,
                               on_epochs=self.choices, prob_bars=self.choices)
 
+        """
+
         def on_test_batch_start(self, trainer, pl_module, batch, batch_idx, dataloader_idx):
             self.make_logging(pl_module, 'on_test_batch_start', 4, on_steps=self.choices,
                               on_epochs=self.choices, prob_bars=self.choices)
-        def on_test_step_end(self, trainer, pl_module, outputs, batch, batch_idx, dataloader_idx):
-            self.make_logging(pl_module, 'on_test_step_end', 5, on_steps=self.choices,
+        """
```

Comment:
pls do not use this king. of commenting, it not clear if you see just a partial diff...